CMake/cdat_modules/scipy_pkg.cmake

```diff
@@ -1,16 +1,16 @@
 set(SCIPY_MAJOR 0)
 set(SCIPY_MINOR 10)
 set(SCIPY_PATCH 0)
-set(SCIPY_MAJOR_SRC 0)
-set(SCIPY_MINOR_SRC 15)
-set(SCIPY_PATCH_SRC 1)
+set(SCIPY_MAJOR_SRC dde07a64407ffaa9442b3d8298c6c26ff91fb384)
```

Comment:
I think a specific version is obtained from SciPy's Github, not just the master at the time of build. See [scipy/scipy:dde07a6](https://github.com/scipy/scipy/tree/dde07a64407ffaa9442b3d8298c6c26ff91fb384)